README.md

```diff
@@ -86,6 +86,7 @@ dizda_cloud_backup:
     processor:
         type: tar # Required: tar|zip|7z
         options:
+            executable: zip|7z # If not added to path variable please set binary path for zip and 7z! e.g. on Windows: 'C:\Program Files\7-Zip\7z.exe'
```

Comment:
I think it would be better to specify the possibilities right in the comment, otherwise it's a little bit confusing.